frontend/src/main/pages/AdminAnnouncementsPage.js

```diff
@@ -36,6 +37,7 @@ export default function AdminAnnouncementsPage() {
           </Row>
         </div>
       </BasicLayout>
+      // Stryker restore all
```

Comment:
```suggestion

```

Reply:
I have also removed this line right now on my local, and I will push it back again later. Really sorry about this problem